crates/biome_js_analyze/src/lint/nursery/use_default_switch_clause.rs

```diff
@@ -0,0 +1,93 @@
+use std::ops::Not;
+
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_js_syntax::JsSwitchStatement;
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Require default cases in switch statements.
+    ///
+    /// Some code conventions require that all switch statements have a default case. The thinking is that it’s better
+    /// to always explicitly state what the default behavior should be so that it’s clear whether or not the developer
+    /// forgot to include the default behavior by mistake.
```

Comment:
```suggestion
    /// Some code conventions require that all switch statements have a default clause. The thinking is that it’s better
    /// to always explicitly state what the default behavior should be so that it’s clear whether or not the developer
    /// forgot to include the default behavior by mistake.
```

Reply:
done!